File: TWASALNI/src/Utilisateurs/UtilisateursBundle/Entity/Courses.php
<?php

namespace Utilisateurs\UtilisateursBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Courses
 *
 * @ORM\Table(name="courses")
 * @ORM\Entity(repositoryClass="Utilisateurs\UtilisateursBundle\Repository\CoursesRepository")
 */
class Courses
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="depart", type="string", length=255)
     */
    private $depart;

    /**
     * @var string
     *
     * @ORM\Column(name="destination", type="string", length=255)
     */
    private $destination;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_course", type="datetime")
     */
    private $dateCourse;

    /**
     * @var float
     *
     * @ORM\Column(name="prix", type="float")
     */
    private $prix;


    /**
     * @ORM\ManyToOne(targetEntity="Utilisateurs")
     * @ORM\JoinColumn(name="client_id",referencedColumnName="id")
     */
    private $client;

    /**
     * @ORM\ManyToOne(targetEntity="Utilisateurs")
     * @ORM\JoinColumn(name="partenaire_id",referencedColumnName="id")
     */
    private $partenaire;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set depart
     *
     * @param string $depart
     *
     * @return Courses
     */
    public function setDepart($depart)
    {
        $this->depart = $depart;

        return $this;
    }

    /**
     * Get depart
     *
     * @return string
     */
    public function getDepart()
    {
        return $this->depart;
    }

    /**
     * Set destination
     *
     * @param string $destination
     *
     * @return Courses
     */
    public function setDestination($destination)
    {
        $this->destination = $destination;

        return $this;
    }

    /**
     * Get destination
     *
     * @return string
     */
    public function getDestination()
    {
        return $this->destination;
    }

    /**
     * Set dateCourse
     *
     * @param \DateTime $dateCourse
     *
     * @return Courses
     */
    public function setDateCourse($dateCourse)
    {
        $this->dateCourse = $dateCourse;

        return $this;
    }

    /**
     * Get dateCourse
     *
     * @return \DateTime
     */
    public function getDateCourse()
    {
        return $this->dateCourse;
    }

    /**
     * Set prix
     *
     * @param float $prix
     *
     * @return Courses
     */
    public function setPrix($prix)
    {
        $this->prix = $prix;

        return $this;
    }

    /**
     * Get prix
     *
     * @return float
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * Set client
     *
     * @param \Utilisateurs\UtilisateursBundle\Entity\Utilisateurs $client
     *
     * @return Courses
     */
    public function setClient(\Utilisateurs\UtilisateursBundle\Entity\Utilisateurs $client = null)
    {
        $this->client = $client;

        return $this;
    }

    /**
     * Get client
     *
     * @return \Utilisateurs\UtilisateursBundle\Entity\Utilisateurs
     */
    public function getClient()
    {
        return $this->client;
    }

    /**
     * Set partenaire
     *
     * @param \Utilisateurs\UtilisateursBundle\Entity\Utilisateurs $partenaire
     *
     * @return Courses
     */
    public function setPartenaire(\Utilisateurs\UtilisateursBundle\Entity\Utilisateurs $partenaire = null)
    {
        $this->partenaire = $partenaire;

        return $this;
    }

    /**
     * Get partenaire
     *
     * @return \Utilisateurs\UtilisateursBundle\Entity\Utilisateurs
     */
    public function getPartenaire()
    {
        return $this->partenaire;
    }
}

File: MOBILE/src/ENTITE/Courses.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ENTITE;

import java.util.Date;




/**
 *
 * @author Arbi
 */
public class Courses {
    private int id;
    private User client;
    private User partenaire;
    private String depart;
    private String destination;
    private Date date_course;
    private float prix;
    private String nompartenaire;
    private String nomclient;
    private int clientId;
    private int partenaireId;

    
    

    public Courses(int id,User client, User partenaire, String depart, String destination, Date date_course, float prix) {
        this.id = id;
        this.client = client;
        this.partenaire = partenaire;
        this.depart = depart;
        this.destination = destination;
        this.date_course = date_course;
        this.prix = prix;
        this.nompartenaire= this.partenaire.getNom();
        this.nomclient= this.client.getNom();
    }

    public Courses(int id, String depart, String destination, float prix) {
        this.id = id;
        this.depart = depart;
        this.destination = destination;
        this.prix = prix;
    }

    public Courses(int id, String depart, String destination, Date date_course, float prix, int clientId, int partenaireId) {
        this.id = id;
        this.depart = depart;
        this.destination = destination;
        this.date_course = date_course;
        this.prix = prix;
        this.clientId = clientId;
        this.partenaireId = partenaireId;
    }

    public Courses() {
    }
    
    

    public String getNomclient() {
        return nomclient;
    }

    public void setNomclient(String nomclient) {
        this.nomclient = nomclient;
    }

    public String getNompartenaire() {
        return nompartenaire;
    }

    public void setNompartenaire(String nompartenaire) {
        this.nompartenaire = nompartenaire;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public User getClient() {
        return client;
    }

    public void setClient(User client) {
        this.client = client;
    }

    public User getPartenaire() {
        return partenaire;
    }

    public void setPartenaire(User partenaire) {
        this.partenaire = partenaire;
    }

    public String getDepart() {
        return depart;
    }

    public void setDepart(String depart) {
        this.depart = depart;
    }

    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }

    public Date getDate_course() {
        return date_course;
    }

    public void setDate_course(Date date_course) {
        this.date_course = date_course;
    }

    public float getPrix() {
        return prix;
    }

    public void setPrix(float prix) {
        this.prix = prix;
    }
    
    
    public int getClientId() {
        return clientId;
    }

    public void setClientId(int clientId) {
        this.clientId = clientId;
    }

    public int getPartenaireId() {
        return partenaireId;
    }

    public void setPartenaireId(int partenaireId) {
        this.partenaireId = partenaireId;
    }

    @Override
    public String toString() {
        return "Courses{" + "id=" + id + ", depart=" + depart + ", destination=" + destination + ", date_course=" + date_course + ", prix=" + prix + ", clientId=" + clientId + ", partenaireId=" + partenaireId + '}';
    }
    
    
    
    
}
